zcash_transparent/src/bundle.rs

```diff
@@ -173,7 +173,7 @@ impl OutPoint {
     fn is_null(&self) -> bool {
         // From `BaseOutPoint::IsNull()` in zcashd:
         //   return (hash.IsNull() && n == (uint32_t) -1);
-        self.hash.as_ref() == &[0; 32] && self.n == u32::MAX
+        self.hash.is_null() && self.n == u32::MAX
```

Comment:
`zcash_transparent` needs to depend on at least `zcash_protocol 0.5.3`, but the workspace that it gets its dependency from allows `zcash_protocol 0.5.1`. Technically we don't need to bump the version for any crate other than `zcash_transparent`, so maybe we just temporarily alter its `Cargo.toml` directly instead of the workspace one (copying in the version and path dep and then bumping the version)? We'd presumably remember to put it back to a workspace dep when we release `zcash_protocol 0.6` as the path dependency in `zcash_transparent` would break.